src/app/containers/App/App.jsx

```diff
@@ -6,22 +6,35 @@
 import { Component } from 'react';
 import { renderRoutes } from 'react-router-config';
 import { withRouter } from 'react-router-dom';
+import getRouteProps from '../../routes/getInitialData/utils/getRouteProps';
 
 import loadInitialData from '../../routes/loadInitialData';
 
 export class App extends Component {
   constructor(props) {
     super(props);
 
+    const { service, isAmp } = getRouteProps(
```

Comment:
This method is called twice in this class. I have written a new issue suggesting we refactor this file in another PR.

The reason for carrying out this refactor in another PR is that this one is already affecting a lot of files and changes up out main data object. carry out a full refactor of the App class now will add unnecessary complication to this PR.

Issue to further refactor this file: https://github.com/bbc/simorgh/issues/1676